cloudformation/tasking-manager.template.js

```diff
@@ -0,0 +1,219 @@
+          }]
+        },
+        InstancesDistribution: {
+          OnDemandAllocationStrategy: 'prioritized',
+          OnDemandBaseCapacity: 0,
+          OnDemandPercentageAboveBaseCapacity: 50,
+          SpotAllocationStrategy: 'lowest-price',
+          SpotInstancePools: 2
+        }
+      }
+    }
+  },
+  TaskingManagerEC2LaunchTemplate: {
+    Type: 'AWS::EC2::LaunchTemplate',
+    Properties: {
+      LaunchTemplateName: cf.join('-', [cf.stackName, 'ec2', 'launch', 'template']),
+      LaunchTemplateData: {
+        UserData: cf.userData([
+          'sudo apt-get update &&',
+          'sudo apt-get -y upgrade &&',
+          'sudo add-apt-repository ppa:jonathonf/python-3.6 &&',
+          'sudo apt-get update &&',
+          'sudo apt-get -y install python3.6 &&',
+          'sudo apt-get -y install python3.6-dev &&',
+          'sudo apt-get -y install python3.6-venv &&',
+          'sudo apt-get -y install curl &&',
+          'curl -sL https://deb.nodesource.com/setup_6.x > install-node6.sh &&',
+          'sudo chmod +x install-node6.sh &&',
+          'sudo ./install-node6.sh &&',
+          'sudo apt-get -y install nodejs &&',
+          'sudo npm install gulp -g &&',
+          'npm i browser-sync --save &&',
+          'sudo apt-get -y install postgresql-9.5 &&',
+          'sudo apt-get -y install libpq-dev &&',
+          'sudo apt-get -y install postgresql-server-dev-9.5 &&',
+          'wget http://postgis.net/stuff/postgis-2.2.4dev.tar.gz &&',
+          'tar -xvzf postgis-2.2.4dev.tar.gz &&',
+          'sudo apt-get -y install libxml2 &&',
+          'sudo apt-get -y install libxml2-dev &&',
+          'sudo apt-get -y install libgeos-3.5.0 &&',
+          'sudo apt-get -y install libgeos-dev &&',
+          'sudo apt-get -y install libproj9 &&',
+          'sudo apt-get -y install libproj-dev &&',
+          'sudo apt-get -y install libgdal1-dev &&',
+          'sudo apt-get -y install libjson-c-dev &&',
+          'cd postgis-2.2.4dev &&',
+          './configure &&',
+          'make &&',
+          'sudo make install &&',
+          'cd .. &&',
+          'sudo apt-get -y install git &&',
+          'git clone --recursive https://github.com/hotosm/tasking-manager.git &&',
+          'cd tasking-manager/ &&',
+          'python3.6 -m venv ./venv &&',
+          '. ./venv/bin/activate &&',
+          'pip install --upgrade pip &&',
+          'pip install -r requirements.txt &&',
+          'echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p &&',
+          'sudo -u postgres psql -c "CREATE USER hottm WITH PASSWORD 'hottm';" &&',
+          'sudo -u postgres createdb -T template0 tasking-manager -E UTF8 -O hottm &&',
+          'sudo -u postgres psql -d tasking-manager -c "CREATE EXTENSION postgis;" &&',
+          'export TM_DB="postgresql://hottm:hottm@localhost/tasking-manager" &&',
+          'export TM_CONSUMER_KEY="VLm4AmuigZODSZQSEEdarv8LhFi4NFodc9JbvvEl" &&',
```

Comment:
@arunasank are these env variables supposed to be here? are they secrets?

Reply:
👋 dakota, squashed and force pushed to fix this! 🌮